cli_helpers/tabular_output/delimited_output_adapter.py

```diff
@@ -11,6 +11,11 @@
 supported_formats = ('csv', 'tsv')
 preprocessors = (override_missing_value, bytes_to_string)
 
+class linewriter():
+    def reset(self):
+        self.lines = []
```

Comment:
Since linewriter requires the `lines` attribute to work, it should create that attribute on instantiation. `reset()` can be used to clear the lines.

Also, in Python 3, classes are always new-style. Just to be safe we should probably make this a new-style class in Python 2: `class linewriter(object):